tests/conftest.py

```diff
@@ -90,6 +91,8 @@ def test_images(request):
         # download test images and other binaries
         checkout_dir = Path(__file__).parents[1] / ".test_images"
 
+        if not checkout_dir.exists() and not use_internet:
+            pytest.skip("Internet use disabled and `.test_images` not found")
```

Comment:
```suggestion
            pytest.skip("Internet use disabled and `.test_images` not found.")
```

Nit - but I like sentences ending with a period.